lib/auditd/common.go

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auditd
+
+import (
+	"github.com/gravitational/trace"
+	"github.com/mdlayher/netlink"
+)
+
+// EventType represent auditd message type.
+// Values comes from https://github.com/torvalds/linux/blob/08145b087e4481458f6075f3af58021a3cf8a940/include/uapi/linux/audit.h#L54
+type EventType int
```

Comment:
I like to see the custom types, nice

Reply:
I was always a huge fan of "strong typdefs" 😄